### PR TITLE
Provide reference link for parameters in config file on node-specific-configurations page

### LINKED ISSUE #2401: Document: refactor: use separate configuration for exchange connection pool for batch and streaming
This issue tracks the documentation update needed for the merged PR #17768.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/17768
Source PR Merged At: 2024-07-23T07:32:59Z

### PR DESCRIPTION
**Pull request by ShanlanLi**
## Info

- **Description**

  - An update for code pr `refactor: use separate configuration for exchange connection pool for batch and streaming`.
  - We don't use a single page to introduce parameters in the configuration file, so provide a reference link in this pr for users.

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - https://github.com/risingwavelabs/risingwave/pull/17768

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/2401

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

File: docs/manage/node-specific-configurations.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ slug: /node-specific-configurations
   <link rel="canonical" href="https://docs.risingwave.com/docs/current/node-specific-configurations/" />
 </head>
 
-In RisingWave, certain configurations are node-specific and can vary between different nodes. These configurations are stored in a TOML file, which is read at system startup.
+In RisingWave, certain configurations are node-specific and can vary between different nodes. These configurations are stored in a [TOML file](https://github.com/risingwavelabs/risingwave/tree/0be5d74f26526bcaa0b6d18a0cedd627d23cedf7/src/config), which is read at system startup.
```
Comment (BugenZhao) [inline docs/manage/node-specific-configurations.md]: Shall we link to the `example.toml` here?

**Review by BugenZhao — COMMENTED** [review]
We use a fixed commit hash (`0be5d74f26526bcaa0b6d18a0cedd627d23cedf7`) in the links. I think it would be better to...

- link to the corresponding release tags
- or always to link to the `HEAD` of main branch

**Comment by ShanlanLi** [comment]
> We use a fixed commit hash (`0be5d74f26526bcaa0b6d18a0cedd627d23cedf7`) in the links. I think it would be better to...
> 
> * link to the corresponding release tags
> * or always to link to the `HEAD` of main branch

Thanks for the useful tip. It helped me discover some points I hadn't noticed before. Have shared this with my colleagues.

**Review by BugenZhao — APPROVED** [review]

File: docs/manage/node-specific-configurations.md [hunk 2]
```diff
@@ -50,6 +50,8 @@ Node-specific configurations can be set in the `risingwave.toml` configuration f
 
 Any items present in `risingwave.toml` will override the default values in the source code. If no configuration file is specified, the default values in `/risingwave/src/common/src/config.rs` will be used.
 
+To know more details about the parameters in the configuration file, you can visit the [RisingWave configuration files directory](https://github.com/risingwavelabs/risingwave/tree/main/src/config). You will find various information there, like the [definitions and default values of these parameters](https://github.com/risingwavelabs/risingwave/blob/main/src/config/docs.md).
```
Comment (WanYixian) [inline docs/manage/node-specific-configurations.md]:
```suggestion
For more details about the parameters in the configuration file, see [RisingWave configuration files directory](https://github.com/risingwavelabs/risingwave/tree/main/src/config). There you'll find information like the [definitions and default values of these parameters](https://github.com/risingwavelabs/risingwave/blob/main/src/config/docs.md).
```

**Review by WanYixian — APPROVED** [review]
The collocation of "various information" is not that common, so revised it.

**Pull request merged**